systemd/fty-proxy.service.in

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Prepare common run-time environment variables for 42ity services
```

Comment:
I think this description should be updated.

Reply:
Yes :)